executor/README.md

```diff
@@ -0,0 +1,270 @@
+Cook Executor
+=============
+
+The cook executor is a custom executor written in python. It replaces the default command executor in order to enable a number of features for both operators and end users.
+
+For Users
+-----
+
+The cook executor supports a number of (completely optional) features beyond the standard Mesos command executor. It allows users to specify additional commands that run either before or after a job's primary command. This enables the use of existing utilities, and promotes sharing and reuse. It also provides an HTTP API that allows commands to expose progress updates and custom failure messages, as well as update environment variables.
+
+### Scheduler HTTP API
```

Comment:
Should this be `Executor HTTP API`? If so, should the endpoints below be `raw-executor`?